scripts/setup_fvp_optee.sh

```diff
@@ -180,6 +180,8 @@ export CROSS_COMPILE=$DST_AARCH64_NONE_GCC/bin/aarch64-none-elf-
 cd $DST_KERNEL
 
 if [ ! -f ".config" ]; then
+	sed -i '/config ARM64$/a\
+	select DMA_SHARED_BUFFER' arch/arm/Kconfig
```

Comment:
This will modify the kernel sources and make them dirty. It would probably better to use merge_config.sh (although not as easy...)